cardano-cli/src/Cardano/CLI/EraBased/Query/Option.hs

```diff
@@ -713,6 +718,11 @@ pQueryDRepStakeDistributionCmd era envCli = do
     QueryDRepStakeDistributionCmdArgs w
       <$> pQueryCommons era envCli
       <*> pAllOrOnlyDRepHashSource
+      <*> pFormatFlags
+        "drep-stake-distribution query output"
```

Comment:
There should be a function that takes a `QueryCmds era` and returns "* query output"